examples/query-log-example-2c.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "2c”,
```

Comment:
Okay, taking a look. Those were from the original repo though. Ouch. According to Matthew we would need to fork this version now ;)